src/NearOrg/CookiePolicy.jsx

```diff
@@ -123,9 +123,8 @@ return (
           content: (
             <>
               <p>
-                <a href="/${REPL_ACCOUNT}/widget/NearOrg.CookieDetails/">
-                  See details on all cookies in use on near.org here
-                </a>
+                See details on all cookies in use on near.org{" "}
+                <a href="/${REPL_ACCOUNT}/widget/NearOrg.CookieDetails/">here</a>.
```

Comment:
It's better to use Link tag here but we can change this after release 
```suggestion
                <Link href="/${REPL_ACCOUNT}/widget/NearOrg.CookieDetails/">here</link>.
```